tests/test_experiment.py

```diff
@@ -229,3 +235,77 @@ def test_telemetry_default(
 def test_error_on_cobalt() -> None:
     with pytest.raises(SSUnsupportedError):
         exp = Experiment("cobalt_exp", launcher="cobalt")
+
+
+def test_default_orch_path(test_dir: str, wlmutils: "conftest.WLMUtils") -> None:
+    """Ensure the default file structure is created for Orchestrator"""
+
+    exp_name = "default-orch-path"
+    exp = Experiment(exp_name, launcher="local", exp_path=test_dir)
+    db = exp.create_database(
+        port=wlmutils.get_test_port(), interface=wlmutils.get_test_interface()
+    )
+    exp.start(db)
+    exp.stop(db)
```

Comment:
Instead of actually starting and stopping the database, you could try monkey patching out the launch of the database.